images/kor/config/template.apko.yaml

```diff
@@ -0,0 +1,16 @@
+contents:
+  packages:
+    - kor
```

Comment:
you have to comment this package here if you are willing to set that through extra_packages

Reply:
But if I use the package here, I think I can keep the `extra_packages` as an empty array, i.e. the current state.
Meaning no need to make this change, and https://github.com/chainguard-images/images/pull/1645/files#r1392497471. Is my understanding correct?

Reply:
not sure about that, lets' give it a try.